MetaMask/metamask-extension
# Security Alert Settings: update copy to match figma

## Explanation



Fixes https://github.com/MetaMask/MetaMask-planning/issues/1064

- update copy based on Figma design (found in related issue)
- additionally, combine the sentences to avoid awkward breakage

## Screenshots/Screencaps



### Before
<img width="957" alt="Screenshot 2023-07-31 at 11 27 28 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/14eb4573-bdde-45e9-a0a9-bc6a75f126c4">



### After without combining text
<img width="1012" alt="Screenshot 2023-07-31 at 11 34 42 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/05e111d6-bff9-4dd8-b5dc-28551a870830">

### After combining text (Final)
<img width="991" alt="Screenshot 2023-07-31 at 11 35 45 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/0acee798-3d68-4726-897a-7e02068d36be">


## Manual Testing Steps

1. Uncomment `:blockaid` line in build.yml to enable Blockaid feature
2. Run Flask version `yarn start:flask`
3. Go to Settings > Experimental
4. Observe UI


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
